Source/Core/Core/Core.h

```diff
@@ -102,7 +102,8 @@ std::string StopMessage(bool main_thread, std::string_view message);
 bool IsRunning();
 bool IsRunningAndStarted();       // is running and the CPU loop has been entered
 bool IsRunningInCurrentThread();  // this tells us whether we are running in the CPU thread.
-bool IsCPUThread();               // this tells us whether we are the CPU thread.
+bool IsBooted();     // whether CPU thread has finished its startup sequence and isn't shutting down
+bool IsCPUThread();  // this tells us whether we are the CPU thread.
```

Comment:
The vertical offset of the comments needs to be adjusted

Reply:
The linter didn't complain.

Reply:
It just looks neat

Reply:
You can't make this look any neater. The `IsBooted` line is already 100 characters long.